cicd-stable.groovy

```diff
@@ -0,0 +1,15 @@
+node('linux') 
+{
+        stage ('Poll') {
+                checkout([
+                        $class: 'GitSCM',
+                        branches: [[name: '*/main']],
+                        doGenerateSubmoduleConfigurations: false,
+                        extensions: [],
+                        userRemoteConfigs: [[url: 'https://github.com/ZOSOpenTools/gitport.git']]])
+        }
+
+        stage('Build') {
+                build job: 'Port-Pipeline', parameters: [string(name: 'PORT_GITHUB_REPO', value: 'https://github.com/ZOSOpenTools/gitport.git'), string(name: 'PORT_DESCRIPTION', value: 'git description' ), string(name: 'BUILD_LINE', value: 'STABLE')]
```

Comment:
Same here

Reply:
Updated in PR: https://github.com/ZOSOpenTools/gitport/pull/89/files